setup.py

```diff
@@ -164,7 +164,7 @@
             'psycopg2-binary>=2.7.3.2,<3.0',
         ],
         prometheus=[
-            'prometheus-client>=0.2.0,<0.5.0,!=0.4.0,!=0.4.1',
+            'prometheus-client==0.6.0',
```

Comment:
Note the comment at the top of setup.py - it is generated from setup.cfg, so that's where this change needs to be made, and run 'make setup.py' to update setup.py.

Reply:
thanks, will take a look

Reply:
I've modified the `setup.cfg` to include `0.6.0`.